app/views/layouts/base.html.erb

```diff
@@ -40,6 +41,19 @@
       document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
     </script>
 
+      <% if try(:current_namespace).eql?('candidate_interface') && cookies['consented-to-apply-cookies'].blank? %>
+        <%= render 'shared/cookies_banner',
+                   cookies_path: candidate_interface_cookies_path,
+                   cookie_preferences_path: candidate_interface_cookie_preferences_path(consent: 'yes'),
+                   service_name: 'Apply for teacher training' %>
+
+      <% elsif try(:current_namespace).eql?('provider_interface') && cookies['consented-to-manage-cookies'].blank? %>
+        <%= render 'shared/cookies_banner',
+                   cookies_path: provider_interface_cookies_path,
+                   cookie_preferences_path: provider_interface_cookie_preferences_path(consent: 'yes'),
+                   service_name: 'Manage teacher training applications' %>
```

Comment:
Is the service name internationalised somewhere?

Reply:
There’s a `service_name` helper, which provides the name of the current service being viewed: https://github.com/DFE-Digital/apply-for-teacher-training/blob/9227dcb8689bdd9390aecafce7f3b51510cb3189/app/helpers/application_helper.rb#L7

Reply:
@paulrobertlloyd thanks! I was not aware of this. I can probably drop the relevant locale entry then!